src/envoy/mixer/grpc_transport.cc

```diff
@@ -110,13 +107,14 @@ void GrpcTransport<RequestType, ResponseType>::Cancel() {
 template <class RequestType, class ResponseType>
 typename GrpcTransport<RequestType, ResponseType>::Func
 GrpcTransport<RequestType, ResponseType>::GetFunc(Upstream::ClusterManager& cm,
+                                                  const std::string& cluster_name,
                                                   const HeaderMap* headers) {
-  return [&cm, headers](const RequestType& request, ResponseType* response,
+  return [&cm, &cluster_name, headers](const RequestType& request, ResponseType* response,
```

Comment:
does `cluster_name` outlive the returned function? If not sure, capture by value.

Reply:
I think it does? Because we used to pass kMixerServerClusterName variable to GrpcTransport.

Reply:
`kMixerServerClusterName` is a global constant so it definitely outlives. But does the cluster_name outlive?